src/memory.cpp

```diff
@@ -239,7 +242,7 @@ uint8_t debug_emu_read(uint8_t reg)
 		case 5: return gif_recorder_get_state();
 		case 6: return wav_recorder_get_state();
 		case 7: return Options.no_keybinds ? 1 : 0;
-		case 8: return (clockticks6502 >> 0) & 0xff;
+		case 8: return (clockticks6502 >> 0) & 0xff;	// don't do snapshotting here because no state should be changed
```

Comment:
One last thing: While this shouldn't actually perform the snapshot, it should return the relevant values of clock_snap, since that's the behavior in the "real" case.

Reply:
True, what @indigodarkwolf is the behavior of x16-emulator master

Reply:
Fortunately, it's an easy tweak, so I've gone ahead and made it.